k8s-operator/api-proxy/proxy.go

```diff
@@ -283,6 +309,100 @@ func (ap *APIServerProxy) sessionForProto(w http.ResponseWriter, r *http.Request
 	ap.rp.ServeHTTP(h, r.WithContext(whoIsKey.WithValue(r.Context(), who)))
 }
 
+func (ap *APIServerProxy) recordRequestAsEvent(req *http.Request, who *apitype.WhoIsResponse) error {
+	failOpen, addrs, err := determineRecorderConfig(who)
+	if err != nil {
+		ap.log.Errorf("error trying to determine whether the kubernetes api request needs to be recorded: %v", err)
```

Comment:
Are we double logging here? It looks like the error returned from `recordRequestAsEvent` is already logged when called above. Should we just wrap and return instead?